Gordon360/Services/NewsService.cs

```diff
@@ -169,7 +169,7 @@ public StudentNews SubmitNews(StudentNews newsItem, string username)
                 var serverAddress = _serverUtils.GetAddress();
                 if (serverAddress is not string) throw new Exception("Could not upload Student News Image: Server Address is null");
```

Comment:
You can remove this since it's been extracted into the `GetImageURL` method.

Reply:
Should I also make the `GetImagePath(string filename)` private the same as `GetImageURL(string filename)`?

Reply:
Yeah, since it's not used outside the service anymore, you can make it private and remove it from the interface.